docs/articles/actors/routers.md

```diff
@@ -96,6 +96,27 @@ Routers are implemented as actors, so a router is supervised by it's parent, and
 
 By default, pool routers use a custom strategy that only returns `Escalate` for all exceptions, the router supervising the failing worker will then escalate to it's own parent, if the parent of the router decides to restart the router, all the pool workers will also be recreated as a result of this.
 
+## Sending messages to actors behind router
```

Comment:
This headline is wrong. It implies that you are explaining how to directly send messages to a Router's `Routees` . This is not true.
Its better to change it to something like: "How routers communicate with their routee's"